e2e/cephfs.go

```diff
@@ -280,6 +280,25 @@ var _ = Describe("cephfs", func() {
 				}
 			})
 
+			// test only if ceph-csi is deployed via helm
+			if helmTest {
```

Comment:
where are we skipping the storageclass and secret creation for cephfs?

Reply:
This is because we are adding deployment of storageclass and secret already while deploying via helm (See the flags [deploy-sc and --deploy-secret](https://github.com/ceph/ceph-csi/blob/ci/centos/mini-e2e-helm-sc.groovy#L175) ) in the job.
So that is why we are skipping the deployment of storageclass and secret again.